internal/cmd/workspaces.go

```diff
@@ -752,3 +753,76 @@ func buildUpdateReq(ctx context.Context, client coder.Client, conf updateConf) (
 	}
 	return &updateReq, nil
 }
+
+func setPolicyTemplate() *cobra.Command {
+	var (
+		ref      string
+		repo     string
+		filepath string
+		dryRun   bool
+	)
+
+	cmd := &cobra.Command{
+		Use:   "policy-template",
+		Short: "Set workspace policy template",
+		Long:  "Set workspace policy template",
```

Comment:
Can we put a note only site admins can do this?

Reply:
✅